### ISSUE TITLE
Interacting with a relay

### ISSUE THREAD

**Issue by mikeriss** [jb55/nostr-js#9]
How do I interact with a relay? I want to send an event

**Comment by ralyodio** [comment]
Yes for a js library the docs here are pretty bad. Need examples. The deno library looks much better. I'm going to use that instead of this half assed code.
